Profiles/ext-Communication.Payload.ContentCodeableConcept.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It seems that this profile has been copied from MP9; this is not the way to go. Instead the profile should be relocated to the nl-core repository (as it apparently is needed in more use cases than present within a single information standard), and in the next release of MP9 and CiO the nl-core dependency should be updated accordingly.